adapters/avail/src/service.rs

```diff
@@ -157,9 +157,8 @@ impl DaService for DaProvider {
     }
 
     // Extract the blob transactions relevant to a particular rollup from a block.
-    // This method is usually (but not always) parameterized by some configuration option,
-    // such as the rollup's namespace on Celestia. If configuration is needed, it should be provided
-    // to the DaProvider struct through its constructor.
+    // NOTE: The avail light client is expected to be run in app specific mode, and hence the 
```

Comment:
Maybe to make it clear that block can contain other transactions, e.g.
`// transactions in the block are already filtered by rollup.`